packages/studio-ui/src/web/views/Nlu/intents/style.scss

```diff
@@ -87,6 +76,18 @@ p.utterance {
       border-radius: 5px;
     }
   }
+
+  // Fix duplicate placeholder
+  [data-slate-leaf='true'] > span > [contenteditable='false'] + span > [contenteditable='false'],
+  [data-slate-leaf='true']
+    > span
+    > [contenteditable='false']
+    + span
+    > [contenteditable='false']
+    + span
+    > [contenteditable='false'] {
+    display: none !important;
+  }
```

Comment:
I totally acknowledge how hacky this looks! It fixes a bug (which already existed) where the placeholder is rendered 3 times on the first line for some reason. I tried to fix it via the `react-slate-placeholder` plugin, but it's undocumented and unmaintained and I didn't have any luck :^\ 
<img width="536" alt="Screen Shot 2022-02-15 at 3 53 51 PM" src="https://user-images.githubusercontent.com/159949/154147883-97258d3f-36dd-4a8c-9665-6cb592a238df.png">